### DISCUSSION TITLE
The console app still terminate when I don't call await scheduler.Shutdown()

### DISCUSSION THREAD

**Discussion by LeewayHsu** [quartznet/quartznet#655] [General] [answered]
### Expected behavior
These days I try to use quartz 3.0.6, and I read the [quick start](https://www.quartz-scheduler.net/documentation/quartz-3.x/quick-start.html). In the section "Starting a Sample Application", it's said "your application will not terminate by default until you call scheduler.Shutdown()". Howerver, when I delete the " await scheduler.Shutdown();", the console app still terminate, which is different from the things in quartz 2. Is the doc wrong or I misunderstand something?

**Comment by lahma** [comment, marked as answer]
You are correct, this behavior has changed in 3.0. I've clarified this in the documentation, thanks for reporting the error.
